src/chains/ethereum/ethereum/src/provider.ts

```diff
@@ -419,10 +418,22 @@ export class EthereumProvider
     }
   };
 
+  /**
+   * Disconnect the provider instance. This will cause the underlying blockchain to be stopped, and any pending
+   * tasks to be rejected. Await the returned Promise to ensure that everything has been cleanly shut down
+   * before terminating the process.
+   * @return Promise<void> - indicating that the provider has been cleanly disconnected
```

Comment:
Maybe something like:


```suggestion
   * @returns Fullfills with `undefined` once the provider has been disconnected.
```

(note that we use `@returns` instead of `@return`)